### PR TITLE
refactor: final renames

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/components/slider/examples/5/index.ts [hunk 1]
```diff
@@ -4,12 +4,12 @@ import {FormControl, ReactiveFormsModule} from '@angular/forms';
...
-import {TuiSliderComponent, TuiSliderKeyStepsDirective} from '@taiga-ui/kit';
+import {TuiSliderComponent, TuiSliderKeySteps} from '@taiga-ui/kit';
 
 @Component({
     standalone: true,
     imports: [
-        TuiSliderKeyStepsDirective,
+        TuiSliderKeySteps,
         ReactiveFormsModule,
         TuiSliderComponent,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/slider/examples/5/index.ts:14]: Why not `TuiSlider` ?

File: projects/demo/src/modules/components/slider/examples/6/index.ts [hunk 2]
```diff
@@ -5,14 +5,14 @@ import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
 import {TUI_FALSE_HANDLER, tuiClamp} from '@taiga-ui/cdk';
 import {TuiButton, TuiHint} from '@taiga-ui/core';
-import {TuiSliderComponent, TuiSliderThumbLabelComponent} from '@taiga-ui/kit';
+import {TuiSliderComponent, TuiSliderThumbLabel} from '@taiga-ui/kit';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/slider/examples/6/index.ts:8]:
```suggestion
import {TuiSlider} from '@taiga-ui/kit';
```

File: projects/demo/src/modules/directives/copy-processor/examples/import/import.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ```ts
-import {TuiCopyProcessorDirective} from '@taiga-ui/core';
+import {TuiCopyProcessor, from '@taiga-ui/core';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/directives/copy-processor/examples/import/import.md:2]:
```suggestion
import {TuiCopyProcessor} from '@taiga-ui/core';
```

File: projects/demo/src/modules/directives/hovered-change/examples/import/import.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 ```ts
-import {TuiHoveredDirective} from '@taiga-ui/cdk';
+import {TuiHovered, from '@taiga-ui/cdk';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/directives/hovered-change/examples/import/import.md:2]:
```suggestion
import {TuiHovered} from '@taiga-ui/cdk';
```

File: projects/demo/src/modules/directives/media/examples/import/import.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
 ```ts
-import {TuiMediaDirective} from '@taiga-ui/cdk';
+import {TuiMedia, from '@taiga-ui/cdk';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/directives/media/examples/import/import.md:2]:
```suggestion
import {TuiMedia} from '@taiga-ui/cdk';
```

File: projects/demo/src/modules/directives/resizer/examples/import/import.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
 ```ts
-import {TuiResizeableDirective, TuiResizerDirective} from '@taiga-ui/cdk';
+import {TuiResizeable, TuiResizer, from '@taiga-ui/cdk';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/directives/resizer/examples/import/import.md:2]:
```suggestion
import {TuiResizeable, TuiResizer} from '@taiga-ui/cdk';
```

File: projects/demo/src/modules/directives/validator/examples/import/import.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
 ```ts
-import {TuiValidatorDirective} from '@taiga-ui/cdk';
+import {TuiValidator, from '@taiga-ui/cdk';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/directives/validator/examples/import/import.md:2]:
```suggestion
import {TuiValidator} from '@taiga-ui/cdk';
```

File: projects/demo/src/modules/directives/value-changes/examples/import/import.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
 ```ts
-import {TuiValueChangesDirective} from '@taiga-ui/cdk';
+import {TuiValueChanges, from '@taiga-ui/cdk';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/directives/value-changes/examples/import/import.md:2]:
```suggestion
import {TuiValueChanges} from '@taiga-ui/cdk';
```

File: projects/kit/components/slider/test/slider-key-steps.spec.ts [hunk 9]
```diff
@@ -4,12 +4,12 @@ import {TestBed} from '@angular/core/testing';
...
 import {NG_EVENT_PLUGINS} from '@taiga-ui/event-plugins';
 import type {TuiKeySteps} from '@taiga-ui/kit';
-import {TuiSliderComponent, TuiSliderKeyStepsDirective} from '@taiga-ui/kit';
+import {TuiSliderComponent, TuiSliderKeySteps} from '@taiga-ui/kit';
 
 describe('TuiSliderKeyStepsDirective', () => {
     @Component({
         standalone: true,
-        imports: [TuiSliderComponent, ReactiveFormsModule, TuiSliderKeyStepsDirective],
+        imports: [TuiSliderComponent, ReactiveFormsModule, TuiSliderKeySteps],
```
Comment (nsbarsukov) [inline projects/kit/components/slider/test/slider-key-steps.spec.ts:12]:
```suggestion
        imports: [TuiSlider, ReactiveFormsModule],
```

File: projects/demo/src/modules/directives/value-changes/examples/import/import.md (hunk 8 above)
Comment (splincode) [reply projects/demo/src/modules/directives/value-changes/examples/import/import.md:2]: fixed

File: projects/kit/components/slider/test/slider-key-steps.spec.ts (hunk 9 above)
Comment (splincode) [reply projects/kit/components/slider/test/slider-key-steps.spec.ts]: fixed

File: projects/demo/src/modules/directives/validator/examples/import/import.md (hunk 7 above)
Comment (splincode) [reply projects/demo/src/modules/directives/validator/examples/import/import.md]: fixed

File: projects/demo/src/modules/directives/media/examples/import/import.md (hunk 5 above)
Comment (splincode) [reply projects/demo/src/modules/directives/media/examples/import/import.md]: fixed

File: projects/demo/src/modules/directives/hovered-change/examples/import/import.md (hunk 4 above)
Comment (splincode) [reply projects/demo/src/modules/directives/hovered-change/examples/import/import.md]: fixed

File: projects/demo/src/modules/directives/copy-processor/examples/import/import.md (hunk 3 above)
Comment (splincode) [reply projects/demo/src/modules/directives/copy-processor/examples/import/import.md]: fixed

File: projects/demo/src/modules/components/slider/examples/5/index.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/slider/examples/5/index.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**